projects/frontegg-app/src/lib/frontegg-app.service.ts

```diff
@@ -107,6 +106,15 @@ export class FronteggAppService {
       },
     ]);
 
+    this.router.events.subscribe((event) => {
+      if(event instanceof NavigationStart) {
+        const authRoutes = this.mapAuthComponents.map(({path}) => path);
+        const curr = authRoutes.find((path) => event.url.split('?')[0] !==`/${path}` && event.url.startsWith(`/${path}`));
+        if(curr){
+          this.router.navigateByUrl(`/${curr}${decodeURIComponent(event.url.split(curr)[1])}`);
```

Comment:
I think this route will couse infinite routing like we had in previous versions

Reply:
As you can see it [here](https://frontegg.atlassian.net/browse/FR-5499?focusedCommentId=11080) - No, It doesn't have it.